doc/classes/AnimatedSprite2D.xml

```diff
@@ -5,34 +5,63 @@
 	</brief_description>
 	<description>
 		[AnimatedSprite2D] is similar to the [Sprite2D] node, except it carries multiple textures as animation frames. Animations are created using a [SpriteFrames] resource, which allows you to import image files (or a folder containing said files) to provide the animation frames for the sprite. The [SpriteFrames] resource can be configured in the editor via the SpriteFrames bottom panel.
-		After setting up [member frames], [method play] may be called. It's also possible to select an [member animation] and toggle [member playing], even within the editor.
-		To pause the current animation, set [member playing] to [code]false[/code]. Alternatively, setting [member speed_scale] to [code]0[/code] also preserves the current frame's elapsed time.
 	</description>
 	<tutorials>
 		<link title="2D Sprite animation">$DOCS_URL/tutorials/2d/2d_sprite_animation.html</link>
 		<link title="2D Dodge The Creeps Demo">https://godotengine.org/asset-library/asset/515</link>
 	</tutorials>
 	<methods>
+		<method name="get_playing_speed" qualifiers="const">
+			<return type="float" />
+			<description>
+				Gets the actual playing speed of current animation or 0 if not playing. This speed is the [member playback_speed] property multiplied by [code]custom_speed[/code] argument specified when calling the [method play] method.
+				This method can be used to determine if the current animation is playing backwards.
+			</description>
+		</method>
+		<method name="is_playing" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if playing an animation.
+			</description>
+		</method>
+		<method name="pause">
+			<return type="void" />
+			<description>
+				Pauses the currently playing animation. The [member frame] will be kept and calling [method play] or [method play_backwards] without arguments will resume the animation.
+				See also [method stop].
+			</description>
+		</method>
 		<method name="play">
 			<return type="void" />
-			<param index="0" name="anim" type="StringName" default="&amp;&quot;&quot;" />
-			<param index="1" name="backwards" type="bool" default="false" />
+			<param index="0" name="name" type="StringName" default="false" />
+			<param index="1" name="custom_speed" type="float" default="1.0" />
+			<param index="2" name="from_end" type="bool" default="false" />
+			<description>
+				Plays the animation with key [param name]. If [param custom_speed] is negative and [param from_end] is [code]true[/code], the animation will play backwards (which is equivalent to calling [method play_backwards]).
+				If this method is called with that same animation [param name], or with no [param name] parameter, the assigned animation will resume playing if it was paused, or restart if it was stopped (see [method stop] for both pause and stop). If the animation was already playing, it will keep playing.
```

Comment:
The description of the `pause()` method says: `The [member frame] will be kept and calling [method play] or [method play_backwards] without arguments will resume the animation.`.

Will there be a restart after `pause()` or not? If the reset occurs only when `stop()` is called, then this remark in `play()` seems superfluous and confusing (as if the reset occurs in `play()`, provided that `stop()` was called before).

Reply:
I agree with you this is a bit odd, AnimationPlayer has had a recent change about it and it should be rewritten.